data/json/items/comestibles/meat_dishes_human.json

```diff
@@ -191,7 +191,15 @@
     "id": "mannwurst_raw",
     "copy-from": "sausage_raw",
     "name": "raw Mannwurst",
-    "description": "A hefty raw 'long-pork' sausage that has been prepared for smoking.",
+    "description": "A hefty raw 'long-pork' sausage that has been prepared for smoking.  Needs to be loaded into a smoking rack to cook it.",
+    "material": "hflesh"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "mann_bratwurst",
+    "copy-from": "bratwurst_sausage",
+    "name": "Mannbrat",
```

Comment:
You might want to change this into Mannwurst, and change Mannwurst into "human sausage" (or something similar). That might have some weirdness with ids, though.

Reply:
Thanks! 
Added mannwurst_cooked.
mannwurst_raw should be a different recipe since it's based on bratwurst_sausage and not on sausage_raw.